content/sql/concepts/commands/rollback/rollback.md

```diff
@@ -0,0 +1,134 @@
+---
+Title: 'Rollback'
+Description: 'Undoes any work performed in the current transaction. It can also be used to undo work performed by in-doubt transactions.'
+Subjects:
+  - 'Data Science'
+Tags:
+  - 'Database'
+  - 'Documentation'
+  - 'Functions'
+  - 'Queries'
+  - 'PostgreSQL'
+  - 'MySQL'
+  - 'SQLite'
+  - 'Tables'
+CatalogContent:
+  - 'learn-sql'
+  - 'paths/analyze-data-with-sql'
+---
+
+The SQL **`rollback`** command undoes work performed in the current **`transaction`** to a **`savepoint`** or to the last **`commit`** to the database. An SQL transaction is a series of one or more SQL statements in the current session.
+
+> **Note:** For info on how to start and commit a transaction, see the [MySQL documentation](https://dev.mysql.com/doc/refman/8.4/en/commit.html).
+
+Therefore, the rollback procedure allows a user to undo the work to a previous point without applying these statements to the database.
+
+## Syntax
+
+```sql
+ROLLBACK [WORK] [TO SAVEPOINT savepoint_name];
+
+| CLAUSE                      | Description                                                                                                                                                                                 |
+| --------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| TO SAVEPOINT savepoint_name | (optional) Resets database to an established savepoint `savepoint_name` within the current transaction. These savepoint names can be created using the command `SAVEPOINT savepoint_name;`. |
+
```

Comment:
```suggestion
```sql
ROLLBACK [WORK] [TO SAVEPOINT savepoint_name];

| CLAUSE                      | Description                                                                                                                                                                                 |
| --------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
| TO SAVEPOINT savepoint_name | (optional) Resets database to an established savepoint `savepoint_name` within the current transaction. These savepoint names can be created using the command `SAVEPOINT savepoint_name;`. |

```
The syntax is written inside the pseudo blocks (```.....```).

Reply:
I'm not following the change. Do you mean that line 30 should include triple backticks to end the block? Or that line 28 should say "pseudo" in place of "sql"